test/Microsoft.AspNetCore.Mvc.Core.Test/Infrastructure/ObjectResultExecutorTest.cs

```diff
@@ -20,37 +20,6 @@ namespace Microsoft.AspNetCore.Mvc.Internal
 {
     public class ObjectResultExecutorTest
     {
-        [Fact]
```

Comment:
I'm having trouble understanding changes to existing tests due to this refactoring. Please restore the old approach so that changes forced by the `src` changes are obvious.